src/Sarif/FileSystem.cs

```diff
@@ -266,5 +266,16 @@ public void DeleteDirectory(string path, bool recursive = false)
         {
             Directory.Delete(path, recursive);
         }
+
+        /// <summary>
+        /// Deletes the specified file.
+        /// </summary>
+        /// <param name="path">
+        /// The name of the file to be deleted. Wildcard characters are not supported.
+        /// </param>
+        public void DeleteFile(string path)
```

Comment:
>DeleteFile [](start = 20, length = 10)

Generally, we want these names to match the corresponding system API. In this case, we have a problem: Delete is shared as a name between multiple types. I suggest that we prefix Delete with that enclosing type name. So this method s/be named 'FileDelete' and the one preceding it s/be named 'DirectoryDelete'

Reply:
Should we rename the method CreateDirectory as well?

---
In reply to: [496997632](https://github.com/microsoft/sarif-sdk/pull/2072#discussion_r496997632) [](ancestors = 496997632)

Reply:
Well, yes, to be consistent,this looks right. I think that also means we need to rename 'Create' to 'FileCreate', since it maps to File.Create?

---
In reply to: [497004102](https://github.com/microsoft/sarif-sdk/pull/2072#discussion_r497004102) [](ancestors = 497004102,496997632)

Reply:
just updated both

---
In reply to: [497005275](https://github.com/microsoft/sarif-sdk/pull/2072#discussion_r497005275) [](ancestors = 497005275,497004102,496997632)